scripts/getReflectTemplatesCacheKey.sh

```diff
@@ -2,13 +2,17 @@
 
 rm -rf prisma-templates-repo
 
-latestSha=$(gh repo clone prisma/templates prisma-templates-repo 2> /dev/null -- --depth 1 && cd prisma-templates-repo && git rev-parse HEAD)
+
+latestSha=$(git clone https://github.com/prisma/templates prisma-templates-repo && cd prisma-templates-repo && git rev-parse HEAD)
```

Comment:
Saw a `gh` auth login error in CI which was probably not visible with `2> /dev/null -- --depth 1`.